crates/bitwarden-json/src/client.rs

```diff
@@ -44,41 +45,43 @@ impl Client {
             }
         };
 
+        let mut client = self.0.lock().await;
```

Comment:
Bit un-intuitive that we never have to release the lock and it's handled on drop.